cmd/kube-proxy/app/server.go

```diff
@@ -205,6 +205,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&o.config.IPVS.StrictARP, "ipvs-strict-arp", o.config.IPVS.StrictARP, "Enable strict ARP by setting arp_ignore to 1 and arp_announce to 2")
 	fs.BoolVar(&o.config.IPTables.MasqueradeAll, "masquerade-all", o.config.IPTables.MasqueradeAll, "If using the pure iptables proxy, SNAT all traffic sent via Service cluster IPs (this not commonly needed)")
+	fs.BoolVar(&o.config.IPTables.NodePortsOnLocalhost, "nodeports-on-localhost", true, "If false kube-proxy will not allow NodePort services can be accessed on localhost.")
```

Comment:
DisableLocalhostNodePorts: `"If true kube-proxy will disallow NodePort services to be accessed via localhost, for modes which previously allowed it"`

Reply:
Thanks, Done

Reply:
Thanks, Done